docs/data_collection.rst

```diff
@@ -78,9 +78,10 @@ Metrics
 -------
 
 Snippet code, which is executed on about:home, sends HTTP requests to a server
-located at https://snippets-stats.mozilla.org whenever an event occurs that we
-would like to measure. These requests are sampled at a rate of 10%, meaning that
-only 10% of the time an event occurs will a request be made.
+located at https://snippets-stats.moz.works and / or
```

Comment:
It depends on  configuration . The first server may redirect to the second server, thus the "and/or"